airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -316,7 +316,7 @@
       },
       "insights_lookback_window": {
         "title": "Insights Lookback Window",
-        "description": "The attribution window",
+        "description": "The attribution window. Facebook freezes insight data 28 days after it was generated, which means that all data from the past 28 days may have changed since we last emitted it, so we retrieve it again. If you set your custom value of lookback window in Facebook account, please provide it here too.",
```

Comment:
```suggestion
        "description": "The attribution window. Facebook freezes insight data 28 days after it was generated, which means that all data from the past 28 days may have changed since we last emitted it, so you can retrieve refreshed insights from the past by setting this parameter. If you set a custom lookback window value in Facebook account, please provide the same value here.",
```